apps/frontend/src/utils/recaptcha/load-google-recaptcha.ts

```diff
@@ -0,0 +1,18 @@
+const reCaptchaSiteKey = process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY;
+
+export default async function loadGoogleRecaptcha(): Promise<void> {
+  return new Promise((resolve) => {
```

Comment:
This function is not protected from race conditions. You should store a promise as a global variable.